nautobot/extras/api/views.py

```diff
@@ -443,7 +444,7 @@ class ImageAttachmentViewSet(ModelViewSet):
 #
 
 
-def _create_schedule(serializer, data, commit, job, job_model, request):
+def _create_schedule(serializer, data, commit, job, job_model, request, celery_queue):
```

Comment:
Any particular reason to only allow sending in the celery queue name instead of opening it to all celery kwargs?

Reply:
The queue is required to create the [`ScheduledJob`](https://github.com/nautobot/nautobot/pull/2321/files#diff-9a2bdd132c5e3a0e6947134b0ae99c641837ad953eb9f5585633bf33e4a60dc9R497) so it made more sense to require the queue explicitly. This should be refactored soon as part of the work coming from #1292 so this method would hopefully go away.